exactonline/api/manager.py

```diff
@@ -60,7 +60,7 @@ def filter(self, **kwargs):
         # kwargs = {'filter': "EntryDate+gt+datetime'2014-01-01'", 'top': 5}
         args = []
         for key, value in kwargs.items():
-            if isinstance(value, unicode):
+            if not isinstance(value, (str, int)):
                 value = value.encode('utf-8')
```

Comment:
This can't possibly be right:

On Python3:

* input `int`, output `unistr`
* input `binstr`, output AttributeError ('bytes' object has no attribute 'encode')
* input `unistr`, output `unistr`

While we in fact, we expect binstrings later on.